server/services/forms/schemas/search.js

```diff
@@ -16,6 +16,7 @@ const search = async (options = {}) => {
     let fields = [];
     const systemConfiguration = await listService.fetch('S_SYSTEM_CONFIGURATION');
     systemConfiguration.profiles.map(profile => {
+        console.log(profile);
```

Comment:
Please don't echo unstructured data to console
```suggestion
```

Reply:
Was for local use, mis-commit. Removed.